src/plugins/kernel_install/utils.sh

```diff
@@ -370,6 +403,10 @@ function install_kernel()
   # Each distro has their own way to update their bootloader
   eval "update_$distro""_boot_loader $name $target $flag"
 
+  # Registering a new kernel
+  cmd="sudo tee -a '$INSTALLED_KERNELS_PATH' > /dev/null"
+  echo "$name" | cmd_manager "$flag" "$cmd"
+
```

Comment:
After reading your explanation about these lines, I think it is a good idea to add a comment that explains these lines (add the shellcheck ref).